runtime/sema/gen/testdata/functions.cdc

```diff
@@ -0,0 +1,16 @@
+pub struct Test {
+    /// This is a test function.
+    pub fun nothing() {}
```

Comment:
Can we make these native functions?

```cadence
pub struct Test {
    pub native fun nothing() {}
}
```
So once empty body support for native functions is added, bodies can also be removed (I guess it's not supported yet?).

Reply:
The `native` modifier can be used, but currently does not have to be used. It makes no semantic difference, because we only use the Cadence definitions for generating types, and not (yet) implementations.

Reply:
yeah, it's more as documentation and for tools (docgen, IDEs, etc.)

Reply:
Yep, we should maybe use it in the actual definition.

Reply:
As for omitting the body: There's currently a bug in the parser, where the . I looked into why, and the problem is that the parsing function for parameter lists, `parseFunctionParameterListAndRest`, skips any trivia , including doc strings, before a possible return type and body, but should actually perform a look-ahead. I'll address this in a separate PR.